### PR TITLE
init Chinese translation

### PR DESCRIPTION
**Pull request by fuergaosi233**
I used a whole day to complete this translation work. This task is too tired.😭😭😭
I have translated English and Chinese. Some parts I think are not suitable for translation. I skipped it.
My operation is completely in accordance with this [url](https://github.com/kubernetes/dashboard/blob/master/docs/developer/internationalization.md)
I refer to the google Chinese document and the Japanese translation in the i18n folder.
I hope he can help more people.
😁😁😁😁

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,7 +113,7 @@
     "hammerjs": "2.0.8",
     "js-yaml": "3.13.1",
     "material-design-icons": "3.0.1",
-    "ng2-ace-editor": "0.3.9",
+    "ng2-ace-editor": "^0.3.9",
```
Comment (floreks) [inline package.json]: This is not related. Please, revert.

**Comment by floreks** [comment]
This is a lot of work. It's awesome Thank you! 

Can you sign the cla with your email used by git?

https://help.github.com/en/articles/setting-your-commit-email-address

File: package.json (hunk 1 above)
Comment (fuergaosi233) [reply package.json]: i run `npm run build` faild beacuse it ,so i reinstall it. I remember `git add` skip this file, maybe i forget it.

Comment (fuergaosi233) [reply package.json]: So do I need to resubmit pr?

**Comment by fuergaosi233** [comment]
I finished the signature

**Comment by fuergaosi233** [comment]
> 这是很多工作。真棒，谢谢！
> 
> 你可以用git使用的电子邮件签名吗？
> 
> https://help.github.com/en/articles/setting-your-commit-email-address

Sorry, I didn't submit the experience of pr to a large project😨. I am now modifying my git email, then what do I need to do?

**Comment by maciaszczykm** [comment]
@fuergaosi233 You will have to create/change your commit to use this email.

**Comment by fuergaosi233** [comment]
> @fuergaosi233 You will have to create/change your commit to use this email.

Restart pr?

**Comment by maciaszczykm** [comment]
@fuergaosi233 You can do this from this branch. PR doesn't matter. It depends on the commit. You can push it again using `git commit --amend` or rebase.

**Comment by floreks** [comment]
Yep `git commit --amend` and `git push -f` should work.

**Comment by fuergaosi233** [comment]
I signed it

**Comment by fuergaosi233** [comment]
I think I finished the signature
why it still needs cla.

**Comment by maciaszczykm** [comment]
/check-cla

@fuergaosi233 Are the emails the same as in the commit? Check with `git log`.

**Comment by fuergaosi233** [comment]
I signed it

**Comment by fuergaosi233** [comment]
I made a mistake in the signature process😓. I am very sorry.

File: package.json (hunk 1 above)
Comment (maciaszczykm) [reply package.json]: No, just revert this change (you can remove the dash manually.

Comment (fuergaosi233) [reply package.json]: 👌🏻

**Comment by floreks** [comment]
Awesome. Everything looks good from my side.

/lgtm

**Comment by maciaszczykm** [comment]
@fuergaosi233 Thanks a lot.

**Comment by idealhack** [comment]
@fuergaosi233 thank you for bringing the Chinese version back
